refinery/core/search_indexes.py

```diff
@@ -65,15 +65,18 @@ def prepare_submitter(self, object):
         submitters = []
 
         for contact in investigation.contact_set.all():
-            submitters.append(contact.last_name + ", " + contact.first_name)
+            submitters.append(
+                "{}, {}".format(contact.last_name, contact.first_name)
```

Comment:
I only saw this in our style guide regarding python string formatting: 
![screen shot 2017-05-08 at 4 56 26 pm](https://cloud.githubusercontent.com/assets/5629547/25825029/56fd0d4a-340f-11e7-9ee9-79ff9a6c0871.png)